### DISCUSSION TITLE
adding domain, or tailscale domain

### DISCUSSION THREAD

**Discussion by texonica** [Freika/dawarich#395] [Q&A] [answered]
hi i'm getting such error when connecting nging proxy manager to dawarich

To allow requests to these hosts, make sure they are valid hostnames (containing only numbers, letters, dashes and dots), then add the following to your environment configuration:
    config.hosts << "dw.aaa.bbb"
    
    in dockge i have
dawarich_app:
...
APPLICATION_HOST: dw.aaa.bbb
      APPLICATION_HOSTS: localhost,dw.aaa.bbb,tailscaledomain.net
      
      and
      
      dawarich_sidekiq:
      ...
APPLICATION_HOST: dw.aaa.bbb
      APPLICATION_HOSTS: localhost,tailscaledomain.net:3009,dw.aaa.bbb
      
      
      
      nor tailscale, nor nginxproxymanager domains work, both get error from ruby

**Comment by Freika** [comment]
I haven't tried to make it work with the tailscale's domains, but here is my approach, hope it will help: https://frey.today/homelab-with-tailscale-docker-caddy-ssl-cloudflare/

**Comment by texonica** [comment]
thank you for converting my issue to a discussion. still this tailscale is "OR". I would like to first get the normal domain working (no tailscale). 
why ruby, which shows that shows that the normal domain doesn't work, nginx proxy manager if it would be working wrongly would break some scripts or websockets...but not get hosts config issue from ruby

**Comment by texonica** [comment]
ok switched to only dw.aaa.bbb in both HOST And HOSTS 
this domain is cloudflare and then nging proxy manager, sadly same issue.

**Comment by texonica** [comment, marked as answer]
ok found a solution, for some reason restarting container in dockge works differently from stop and starting the container. when i did the latter it worked. so changing config should end with stopping and starting container, not just restarting
